lib/consoletest.pm

```diff
@@ -51,6 +58,31 @@ sub post_fail_hook {
     $self->export_logs_desktop;
 }
 
+=head2 get_avc_selinux_alerts
+
+List AVCs that have been recorded during a runtime of a test module that executes this function
+
+=cut
+
+sub get_avc_selinux_alerts {
+    if ((current_console() !~ /root|log/) || (script_run('test -f /var/log/audit/audit.log') != 0)) {
+        return undef;
+    }
+
+    my @logged = split(/\n/, script_output('ausearch -m avc -r', proceed_on_failure => 1));
```

Comment:
You could do `ausearch -m avc -r | tail -n +${avc_record{start}}` here to skip the messages earlier in the flow.

Reply:
That would indeed work, but I still find it useful to see the whole output of the `ausearch` if something would go wrong. e.i. a bug will be introduced to the utility itself etc.